lib/KettleServer.js

```diff
@@ -0,0 +1,303 @@
+/*
```

Comment:
There are multiple failure cases in this file that don't appear to be tested.  Suggest running `istanbul cover tests/all-tests.js` and working through the output.  There are lots of failure cases that are not tested, and not just in this file.